openadapt/crud.py

```diff
@@ -109,28 +116,133 @@ def insert_recording(recording_data):
 
 
 def get_latest_recording():
-    return (
-        db
-        .query(Recording)
-        .order_by(sa.desc(Recording.timestamp))
-        .limit(1)
-        .first()
+    return db.query(Recording).order_by(sa.desc(Recording.timestamp)).limit(1).first()
```

Comment:
@Mustaballer I much prefer the previous formatting. Is there some way to change the configuration so that the previous formatting is kept?

Upon further review it seems that this is supported out of the box by Black:

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#call-chains

Further reading:

https://github.com/psf/black/issues/571

Reply:
I was able to ignore black formatting into one line by lowering the word line max when running black
`black -l 60`

Reply:
resolved in https://github.com/MLDSAI/OpenAdapt/pull/223/commits/4c59c10aeeff5abc7e93c11c9a40bdfbfecebcf9